config/v2/config.go

```diff
@@ -8,101 +8,116 @@ import (
 
 	v1 "github.com/chanzuckerberg/fogg/config/v1"
 	"github.com/chanzuckerberg/fogg/errs"
+	"gopkg.in/yaml.v2"
 )
 
 func ReadConfig(b []byte) (*Config, error) {
 	c := &Config{
 		Docker: false,
 	}
-	e := json.Unmarshal(b, c)
+	e := yaml.Unmarshal(b, c)
+	fmt.Println(string(b))
+	if e != nil {
+		return nil, errs.WrapUser(e, "unable to parse yaml config file")
+	}
+	return c, nil
+}
 
-	return c, errs.WrapUser(e, "unable to parse config")
+func ReadJsonConfig(b []byte) (*Config, error) {
+	c := &Config{
+		Docker: false,
```

Comment:
seems like some are `docker: true` and others `docker: false` is there a reason for the inconsistency?